src/einsteinpy/symbolic/ricci.py

```diff
@@ -141,21 +141,24 @@ def change_config(self, newconfig="ul", metric=None):
         return new_obj
 
 
-class RicciScalar:
+class RicciScalar(BaseRelativityTensor):
     """
+    Inherits from ~einsteinpy.symbolic.tensor.BaseRelativityTensor
     Class for defining Ricci Scalar
     """
 
-    def __init__(self, expression, syms, parent_metric=None):
+    def __init__(self, arr, syms, config="l", parent_metric=None):
```

Comment:
remove the `config` parameter.
`config` is going to be an empty string given scalar is a zero order tensor.